ota-plus-web/docs/modules/ROOT/pages/add-members.adoc

```diff
@@ -0,0 +1,20 @@
+= Add members to environments (Beta)
```

Comment:
```suggestion
= Add members to environments
```

The beta tag will be on the whole category, no need to put it on every page. (Please also remove from the other page titles, and leave it only on the top-level title in the TOC.)